assets/app/scripts/directives/catalog.js

```diff
@@ -1,66 +1,49 @@
 'use strict';
 
 angular.module('openshiftConsole')
-  .directive('catalogTemplate', function($location) {
+  .directive('catalogTemplate', function($location, createFromTemplateURLFilter) {
     return {
       restrict: 'E',
+      // Replace the catalog-template element so that the tiles are all equal height as flexbox items.
+      // Otherwise, you have to add the CSS tile classes to catalog-template.
+      replace: true,
       scope: {
         template: '=',
         project: '='
       },
       templateUrl: 'views/catalog/_template.html',
       link: function(scope, elem, attrs) {
-        $(".select-template", elem).click(function() {
-          // Must trigger off of the modal's hidden event to guarantee modal has finished closing before switching screens
-          $(".modal", elem).on('hidden.bs.modal', function () {
-            scope.$apply(function() {
-              var createURI = URI.expand("project/{project}/create/fromtemplate{?q*}", {
-                project: scope.project,
-                q: {
-                  name: scope.template.metadata.name,
-                  namespace: scope.template.metadata.namespace
-                }
-              });
-              $location.url(createURI.toString());
-            });
-          })
-          .modal('hide');
-
+        elem.click(function() {
```

Comment:
can we use `class="tile-click"` with an `<a href="..." class="tile-target">` inside like we do in other places?